sd-boot/pkg.yaml

```diff
@@ -21,16 +24,11 @@ steps:
         sha256: "{{ .systemd_sha256 }}"
         sha512: "{{ .systemd_sha512 }}"
     env:
-      LD_LIBRARY_PATH: "/toolchain/lib"
       SOURCE_DATE_EPOCH: {{ .BUILD_ARG_SOURCE_DATE_EPOCH }}
+  - network: default
     prepare:
       - |
-        tar -xzf systemd.tar.gz --strip-components=1
-
-        ln -s /toolchain/bin/echo /toolchain/bin/getent
-        mkdir -p /usr/bin
-        ln -sf /toolchain/bin/env /usr/bin/env
-        ln -sf /toolchain/bin/python3 /toolchain/bin/python
+        tar -xzf /tmp/build/0/systemd.tar.gz --strip-components=1
```

Comment:
I think we should never hardcode `/tmp/build/0`, this is internal detail.

Reply:
will do /tmp/scr or sth like that, as in other packages